docs/Developers/Newcomers_Guide.rst

```diff
@@ -203,6 +205,9 @@ into your bash. This will analyze your code and help you fix it.
 Step 5. Sending Your Changes
 ----------------------------
 
+Before starting to write your first commit, check out this link:
+`Writing good commits <http://coala.io/commit>`_.
```

Comment:
if you announce it as a link you should give a link, just embed it into the sentence properly please